src/RemoteTech/Modules/ProtoSignalProcessor.cs

```diff
@@ -25,16 +26,18 @@ public ProtoSignalProcessor(ProtoPartModuleSnapshot ppms, Vessel v)
             mVessel = v;
             Powered = ppms.GetBool("IsRTPowered");
 
+	    int commandMinCrew = DEFAULT_COMMAND_MINIMUM_CREW;
+			
             try {
-                IsCommandStation = Powered && v.HasCommandStation() && v.GetVesselCrew().Count >= ppms.GetInt("RTCommandMinCrew");
+                commandMinCrew = ppms.GetInt("RTCommandMinCrew");
                 RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2}/{3})", 
-                    Powered, v.HasCommandStation(), v.GetVesselCrew().Count, ppms.GetInt("RTCommandMinCrew"));
+                    Powered, v.HasCommandStation(), v.GetVesselCrew().Count, commandMinCrew));
             } catch (ArgumentException) {
                 // I'm assuming this would get thrown by ppms.GetInt()... do the other functions have an exception spec?
-                IsCommandStation = false;
-                RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2})", 
+                RTLog.Notify("ProtoSignalProcessor(Powered: {0}, HasCommandStation: {1}, Crew: {2}/6 [Default]", 
```

Comment:
Oh, good catch. 

I don't really know the API or the code, by 'out of the cfg' do you meansomething other than ppms.GetInt() ?